decidim-admin/app/forms/decidim/admin/component_form.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # A form object used to create participatory processes from the admin
+    # dashboard.
+    #
+    class ComponentForm < Decidim::Form
+      include TranslatableAttributes
+
+      mimic :component
+
+      translatable_attribute :name, String
+      attribute :step_id, Integer
```

Comment:
Because it shouldn't be its job to convert from user input to models, right? How would you implement it?